deprecation.py

```diff
@@ -157,20 +160,23 @@ def _function_wrapper(function):
             # of the parts.
             parts = {
                 "deprecated_in":
-                    " in %s" % deprecated_in if deprecated_in else "",
+                    " %s" % deprecated_in if deprecated_in else "",
                 "removed_in":
-                    ", to be removed in %s" % removed_in if removed_in else "",
-                "period":
-                    "." if deprecated_in or removed_in or details else "",
+                    "\n   To be removed in %s." %
```

Comment:
Now that the deprecated and removed parts are in two different sentences, what about changing "To be removed in" to "This will be removed in"? It read ok before when it was one sentence split by a comma, but I think it could be better now that it's two sentences.

Reply:
Just a second!